data/h3_raster_importer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM perrygeo/gdal-base
+
+RUN pip install --upgrade --no-cache-dir pip
+RUN pip install --no-cache-dir numpy pandas rasterio h3ronpy psycopg2-binary docopt
```

Comment:
every time you install python dependencies without pinning the versions, you-in-6-months kills a kitty because a new version (with breaking changes) was introduced, and your code magically stopped working.